Office 365/o365/ingest/parser.yml

```diff
@@ -569,6 +569,14 @@ stages:
           source.user.email: "{{parse_data.ParsedData.f3u}}"
         filter: "{{json_event.message.Name == 'Mass download by a single user'}}"
 
+      - set:
+          office365.alert.entity_type: "{{parse_data.ParsedData.etype}}"
+          email.message_id: "{{parse_data.ParsedData.imsgid}}"
+          email.cc.address: "{{parse_data.ParsedData.cc.split(',')}}"
+          email.to.address: "{{parse_data.ParsedData.to.split(',') or parse_data.ParsedData.trc.split(',')}}"
+          email.from.address: "{{parse_data.ParsedData.mfrm.split(',') or parse_data.ParsedData.tsd.split(',')}}"
```

Comment:
`trc` and `tsd` seems to not be extracted in the test (`security_compliance_alert_3.json` and `security_compliance_alert4.json`)